protocol/0069-VCBS-validators_chosen_by_stake.md

```diff
@@ -336,6 +336,43 @@ See [limited network life spec](../non-protocol-specs/0005-NP-LIMN-limited_netwo
   * Announce 2 new nodes but self-delegate only to one of them. 
   * Verify that, after 1000 blocks and on the following epoch, only the validator to which we self-delegated got promoted and we now have 4 Tendermint validators and 1 pending validator. 
 
+11. Change ownstake requirement (<a name="0069-VCBS-050" href="#0069-VCBS-050">0069-VCBS-050</a>)
+  * In the same epoch, change the network parameter <reward.staking.delegation.minimumValidatorStake> in a way that 3 validators and three Ersatzvalidators drop below the ownstake requirement, and change the delegation so that 4 (not affected) Ersatzvalidators have a higher score than two (not affected) Validators. Also, give one of the Ersatzvalidators with insufficient ownstake the highest ValidatorScore of all Ersatzvalidators. 
+  * Verify that the the three Ersatzvalidators with now insufficient ownstake are removed at the end of the epoch, and that the three validators with insufficient ownstake are demoted within three epochs (and immediatelly demoted again from being Ersatzvalidators), and that in the fourth epoch the validator with the lowest ValidatorScore is demoted (and each time, the Ersatzvalidatror with the highest score is promoted). Also, verify that the Ersatzvalidator with the highest score and insufficient ownscore got demoted from being an Ersatzvalidator rather than promoted to be a Validator. 
+  
+ 11 (Alternative)
+ 11.a Setup a network with 5 Tendermint validators. (3 validators, 2 ersatzvalidators). In one epoch,
+
+- one ersatzvalidator gets the highest delegated stake, but insufficient ownstake (delegates: 10000)
+- 2 validators drop below ownstake, but have relative high delegated stake (7000)
+-1 validator drops to the lowest delegated stake (1000)
+- 1 ersatzvalidator has 6000 stake and sufficient ownstake
+
+Verify that the the first ersatzvalidator is removed in the epoch change, and one validator with insufficient ownstake is replaced
```

Comment:
What does "removed" mean here? Demoted to pending? or kicked out entirely?

Reply:
Clarified that, it's removed entirely